src/commands/create.js

```diff
@@ -85,8 +87,25 @@ export default asyncCommand({
 		}
 		catch (err) {}
 
+		if (exists && argv.force) {
+			const question = {
+        type: 'confirm',
+        name: 'enableForce',
+        message: `You are using '--force'. Do you wish to continue?`,
+        default: true,
```

Comment:
Okay, will change it once I reach home.

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_